conf/properties.properties

```diff
@@ -21,7 +21,7 @@ DB_PORT = 5432
 
 # Browser that will be used (needs to be installed and PATH set properly, see also README.md )
 # Firefox|Chrome
-BROWSER = Firefox
+BROWSER = Chrome
 
 BROWSER_WIDTH = 1920
 BROWSER_HEIGHT = 1080
```

Comment:
properties.properties should not have any modifications - please revert to current properties.properties in 'master'